govcd/api_vcd_test.go

```diff
@@ -189,6 +189,7 @@ type TestConfig struct {
 			ExternalNetwork           string `yaml:"externalNetwork"`
 			EdgeGateway               string `yaml:"edgeGateway"`
 			NsxtImportSegment         string `yaml:"nsxtImportSegment"`
+			NsxtImportSegment2        string `yaml:"nsxtImportSegment2"`
```

Comment:
Please add this field to `sample_govcd_test_config.yaml`

Reply:
Thanks!. I added to both now - govcd and Terraform as both were missing